index.html

```diff
@@ -91,8 +92,8 @@
   <div class="intro">
     <h1>Type Language Differ</h1>
     <p>Welcome to the online
-      <a href="https://core.telegram.org/mtproto/TL">Type Language</a>
-      <a href="https://en.wikipedia.org/wiki/File_comparison">differ</a>!</p>
+      <a href="https://core.telegram.org/mtproto/TL" target="_blank">Type Language</a>
```

Comment:
I really dislike this. I don't need to tell the user how to open the links. They can middle click or control+click if they want this behaviour.

Reply:
I see
In my scenario, i was expecting the repo to open on diff tab while reading home page along with html
Should i revert it?

Reply:
Yes.